app/helpers/flyover_comments/comments_helper.rb

```diff
@@ -1,6 +1,14 @@
 module FlyoverComments
   module CommentsHelper
 
+    def flyover_comment_content(comment)
+      if comment.deleted_at.nil?
+        comment.content
+      else
+        "Commented deleted on #{comment.deleted_at.to_s(:normal)}"
```

Comment:
Yes translation. If you just put it in the normal translation file, we could override it in a host app translation file with the same key